tso.md

```diff
@@ -0,0 +1,87 @@
+---
+title: TimeStamp Oracle
+summary: Learn about TSO timestamps in TiDB.
+---
+
+One of the tasks of the Placement Driver (PD) is to hand out timestamps to other components of the cluster. Transactions and data get timestamps assigned and this allows the [Percolator](https://research.google.com/pubs/pub36726.html) model in TiDB to work, which is used for MVCC and [transactions](/transaction-overview.md).
```

Comment:
Not sure we also should mention the tso micro service, since PD can now work as separated micro services? Ref: https://github.com/tikv/pd/issues/5836

Reply:
Not sure how visible the TSO micro service is or is going to be. I think it is best for now to merge this as-is and later add TSO micro service things if more of this is clear.